### PR TITLE
Fixing Azure Pipelines url in readme

### PR DESCRIPTION
**Pull request by chrisrpatterson**
Minor update to readme to use correct Azure Pipelines URL

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 [![CircleCI Build Status](https://circleci.com/gh/electron/electron/tree/master.svg?style=shield)](https://circleci.com/gh/electron/electron/tree/master)
 [![AppVeyor Build Status](https://windows-ci.electronjs.org/api/projects/status/nilyf07hcef14dvj/branch/master?svg=true)](https://windows-ci.electronjs.org/project/AppVeyor/electron/branch/master)
-[![Azure Pipelines Build Status](https://github.visualstudio.com/electron/_apis/build/status/electron-builds/electron-osx-testing?branchName=master)](https://github.visualstudio.com/electron/_build/latest?definitionId=36)
+[![Azure Pipelines Build Status](https://dev.azure.com/github/electron/_apis/build/status/electron-builds/electron-osx-testing?branchName=master)](https://github.visualstudio.com/electron/_build/latest?definitionId=36)
```
Comment (jkleinsc) [inline README.md]: @chrisrpatterson are all the URLS changing for Azure Devops?  If so, should we also change the reference to `https://github.visualstudio.com/electron/_build/latest?definitionId=36`?

Comment (shiftkey) [reply README.md]: @jkleinsc I think they'll support both for a while, but it does make sense to rename that URL too while we're in here

**Comment by codebytere** [comment]
@chrisrpatterson could you change the other url @shiftkey referenced above as well? Once that's done we can get this across the finish line!

**Comment by codebytere** [comment]
Closing as we have since moved away from VSTS

**Pull request closed without merging**